tests/ui/proc-macro/no-mangle-in-proc-macro-issue-111888.rs

```diff
@@ -0,0 +1,20 @@
+// run-pass
```

Comment:
I think this needs to be `build-pass` since this is a proc-macro and not directly executable. 

```suggestion
// build-pass
```